containers/dev-requirements.txt

```diff
@@ -0,0 +1,10 @@
+-r requirements.txt
+
+nose-capturestderr
+nose-randomly
```

Comment:
nose-* packages are not needed anymore, because we started to use pytest for unit testing.